ui/tests/acceptance/pki/pki-engine-workflow-test.js

```diff
@@ -25,12 +25,14 @@ module('Acceptance | pki workflow', function (hooks) {
   setupApplicationTest(hooks);
 
   hooks.beforeEach(async function () {
+    this.store = this.owner.lookup('service:store');
     await authPage.login();
     // Setup PKI engine
     const mountPath = `pki-workflow-${uuidv4()}`;
     await enablePage.enable('pki', mountPath);
     this.mountPath = mountPath;
     await logout.visit();
+    clearRecords(this.store);
   });
 
   hooks.afterEach(async function () {
```

Comment:
Line 63 below here is also an issue because there is no `this.pkiToken` for this test - that's only done in the nested modules.

Reply:
I think that caused this to log int with the `root` token because `this.pkiToken` is undefined in this test.

Reply:
weird, I definitely fixed that at some point. Probably stashed somewhere... Thanks for calling it out!